test_my_code.py

```diff
@@ -0,0 +1,15 @@
+import pytest
```

Comment:
This looks like a really useful learning file, but not one relevant to our codebase! We'll replace these with more relevant tests in M3 anyway, so I probably suggest leaving it for now and removing it then, but generally we want to keep this codebase tightly focused on the project otherwise it'll quickly get harder to navigate